assets/src/apps/delivery/Delivery.tsx

```diff
@@ -158,7 +158,7 @@ const Delivery: React.FC<DeliveryProps> = ({
   const { width: windowWidth } = useWindowSize();
   const isLessonEnded = useSelector(selectLessonEnd);
   return (
-    <div className={parentDivClasses.join(' ')}>
+    <div className={`${parentDivClasses.join(' ')} ${localStorage?.theme || ''}`}>
```

Comment:
I check and found that Torus saves a `theme` key in` local storage` that contains the value of the theme student selected on the attempt page so I am just passing adding that in the class list so that external CSS can target this and add their CSS etc.